includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -848,6 +848,14 @@ public static function store_validation_errors( $validation_errors, $url, $args
 			// Note that json_encode() is being used here because wp_slash() will coerce scalar values to strings.
 			update_post_meta( $post_id, '_amp_stylesheets', wp_slash( wp_json_encode( $args['stylesheets'] ) ) );
 		}
+		if ( isset( $args['php_fatal_error'] ) ) {
+			if ( empty( $args['php_fatal_error'] ) ) {
+				delete_post_meta( $post_id, '_amp_php_fatal_error' );
+			} else {
+				// Note that json_encode() is being used here because wp_slash() will coerce scalar values to strings.
+				update_post_meta( $post_id, '_amp_php_fatal_error', wp_slash( wp_json_encode( $args['php_fatal_error'] ) ) );
+			}
+		}
```

Comment:
Note the data model for `amp_validated_url` posts needs to be revisited, but now's not the time. For this reason, I've kept the existing scheme of adding more data to postmeta, and making the changes in various places when in the future it should be centralized in a single model.